index.html

```diff
@@ -148,6 +148,45 @@ <h1>Home</h1>
                     </li>
                 </ul>
             </section>
+            <nav class="bottom-nav">
+                <ul class="grid-auto__columns">
```

Comment:
Very good. I like the way you have another class=""bottom-nav" for responsiveness